requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy==1.22.3;python_version>="3.8"
 numpy==1.21.6;python_version<"3.8"
 scipy==1.8.0;python_version>="3.8"
 scipy==1.7.3;python_version<"3.8"
-networkx==2.4
+networkx>=2.4
```

Comment:
conversely you want the versions in `requirements.txt` to be pinned for reproducibility. We should also pin `dwave-networkx` while we're at it.

If you want to test against a wide dependency range, we should do that in CI.